config/jobs/image-pushing/k8s-staging-sig-storage.sh

```diff
@@ -36,7 +36,6 @@ readonly REPOS=(
 # don't.
 readonly BROKEN_REPOS=(
     kubernetes-csi/csi-driver-iscsi
-    kubernetes-csi/csi-driver-nfs
     kubernetes-csi/csi-proxy
     kubernetes-sigs/container-object-storage-interface-api
 )
```

Comment:
@andyzhangx Should we have `kubernetes-csi/csi-driver-smb` here? The repo currently does not have the required `cloudbuild.yaml`

Reply:
no, should merge release-tools first and then enable `kubernetes-csi/csi-driver-smb`